java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -896,7 +901,8 @@ private static Object[] streamCommands(BaseTransaction<?> transaction) {
                     "0-1", // xadd(streamKey1, Map.of("field1", "value1"), ... .id("0-1").build());
                     "0-2", // xadd(streamKey1, Map.of("field2", "value2"), ... .id("0-2").build());
                     "0-3", // xadd(streamKey1, Map.of("field3", "value3"), ... .id("0-3").build());
-                    3L, // xlen(streamKey1)
+                    "0-4", // xadd(streamKey4, new String[][] {{"field4", "value4"}, {"field4", "value5"}}), ... .id("0-4").build());
+                    3L, // xlen(streamKey4)
```

Comment:
Looks like the xlen above is still passing streamKey1 (line 831)
```suggestion
                    3L, // xlen(streamKey1)
```